File: Calculator VB/Calculator/CalculatorRPN.vb

Partial Public Class CalculatorRPN
    Inherits CalculatorBase

    Public Sub New()
        InitializeComponent()
    End Sub

    Private Sub enter_Click(ByVal sender As Object, ByVal e As EventArgs) Handles enter.Click
        TransferNumberToExpression()
    End Sub

    Private Sub evaluate_Click(ByVal sender As Object, ByVal e As EventArgs) Handles evaluate.Click
        Try
            DisplayResult(Calculator.EvaluateTokensAsRPN())
        Catch __unusedException1__ As Exception
            numericDisplay.Text = "Error!"
        End Try
    End Sub
End Class

File: Calculator/Calculator/CalculatorRPN.cs
﻿using System;

namespace Calculator
{
    public partial class CalculatorRPN : CalculatorBase
    {
        public CalculatorRPN() : base()
        {
            InitializeComponent();
        }

        private void enter_Click(object sender, EventArgs e)
        {
            TransferNumberToExpression();
        } 

        private void evaluate_Click(object sender, EventArgs e)
        {
            try
            {
                DisplayResult(Calculator.EvaluateTokensAsRPN());
            }
            catch (Exception)
            {
                numericDisplay.Text = "Error!";
            }
        }   
    }
}
